src/Chemistry.cpp

```diff
@@ -0,0 +1,19 @@
+//==============================================================================
+// TwoMomentRad - a radiation transport library for patch-based AMR codes
+// Copyright 2020 Benjamin Wibking.
+// Released under the MIT license. See LICENSE file included in the GitHub repo.
+//==============================================================================
+/// \file Chemistry.cpp
+/// \brief Implements methods for primordial chemistry using Microphysics
+///
+
+#include "Chemistry.hpp"
+#include "burn_type.H"
+#include "burner.H"
+
+namespace quokka::chemistry
+{
+
+void chemburner(burn_t chemstate, const Real dt) { return burner(chemstate, dt); }
```

Comment:
You need to add `AMREX_GPU_DEVICE` before `void chemburner(...)` so that it can be called on the GPU.